addons/event_booth_sale/tests/test_event_booth_sale.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime, timedelta
+
+from odoo import fields
+from odoo import Command
+from odoo.addons.event_booth_sale.tests.common import TestEventBoothSaleCommon
+from odoo.tests.common import users
+from odoo.tools import float_compare, float_round
+
+
+class TestEventBoothSale(TestEventBoothSaleCommon):
```

Comment:
Does not seem to hold any test for ``price_reduce_taxinc`` ? I was expecting this new field to be tested, especially since we target a stable version.